packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -183,5 +215,12 @@ export default {
         }).map(() => defaultCardGroupItem),
       }),
     },
+    decorators: [
+      story => html`
+        <div class="dds-ce-demo-devenv--simple-grid--stretch">
```

Comment:
There is no `dds-ce-demo-devenv--simple-grid--stretch` class defined. Did you mean `dds-ce-demo-devenv--grid--stretch`?

Reply:
yup updated!